holmes/core/toolset_manager.py

```diff
@@ -242,13 +261,23 @@ def load_toolset_with_status(
                     cached_status.get("type", ToolsetType.BUILTIN)
                 )
                 toolset.path = cached_status.get("path", None)
-            # check prerequisites for only enabled toolset when the toolset is loaded from cache
+            # check prerequisites for only enabled toolset when the toolset is loaded from cache. When the toolset is
+            # not loaded from cache, the prerequisites are checked in the refresh_toolset_status method.
             if (
                 toolset.enabled
                 and toolset.status == ToolsetStatusEnum.ENABLED
                 and using_cached
             ):
-                toolset.check_prerequisites()  # type: ignore
+                enabled_toolsets_from_cache.append(toolset)
+        # we don't log the status of toolsets when it's loaded from cache, considering
```

Comment:
Why do we need to check prerequisites at all in this case? Isn't the purpose of the cache to avoid checking prerequisites altogether?

If we are checking them, I think we should continue to log. 

For me, the real solution is to avoid to check them altogether when possible, not silencing the log.